pages/src/source/workers/undertaker.md

```diff
@@ -14,7 +14,13 @@ After a short time, graves will decay and eventually disappear. There are [resea
 
 The grave (decayed or not) will hold all the items the citizen had in their inventory at the time of death. The player can right-click the grave to open its inventory and retrieve the items. The Undertaker will walk (run if you've completed the relevant research) toward the grave and retrieve its inventory, then go back to the Graveyard.
 
-Once the Undertaker gets to the Graveyard, they will attempt to revive the deceased citizen. The chance for them to succeed can be increased by researches, the Undertaker's Mana skill, and the level of the Graveyard. The chance is capped at about 10%.
+Once the Undertaker gets to the Graveyard, they will attempt to revive the deceased citizen. The chance for them to succeed can be increased by:
+- researches (1x 1%,1x 2%)
+- the Undertaker's Mana skill (0.125% per Mana Skill Point)
+- the level of the Graveyard (0.5% per Level)
+- the use of totems unlocked by research (Totem gets used up with a chance of 1%)
+
+By default, the chance of reviving can not be greater than 2.5%. This value can be boosted with upgrading the [Mystical Site](../../source/buildings/mysticalsite) (0.5% per Level) and the use of totems (5% for 1 Totem, 7.5% for multiple totems). In total, the maximum chance is 12.5% resp. 1/8.
```

Comment:
I think "This value can be boosted..." should be changed to "This cap can be boosted..."

Can remove the "resp. 1/8" part; just keep the percentages for consistency.

Reply:
That was my mistake; I misread it. It is a cap, so you can revert that change.

Minor note, "cannot" is one word.